web-ui/src/test/java/com/tutrit/config/MySpringContext.java

```diff
@@ -0,0 +1,16 @@
+package com.tutrit.config;
```

Comment:
Take a look at Vlad's MR and suggestions I gave him regarding that .

Reply:
I've renamed class